book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,472 @@
+
+It can be helpful in your communications to state who will be in attendance.
+For example "researchers, clinicians, policymakers, funding bodies, and people with lived experience". 
+
+Finally, consider including short introductions and background information about the speakers and session facilitators in your event information. 
+Many people with lived experience may feel more comfortable engaging in an event where they know who they are speaking with and it can be a helpful way to build rapport. 
+
+(cl-inclusive-events-comms-presentation-material)=
+## Presentation Slides and Event Content
+Although it is common in research to prepare presentation slides at the penultimate hour, to be as inclusive as possible, please work with your speakers and presenters to finalize session content before the event. 
+
+By working with speakers and requesting slides and presentation content ahead of the event, you will be best set up to support the involvement of people with disabilities and lived experience at your event. 
+
+This can help in two core ways, 1) supporting speaking in developing both psychologically and physically accessible content, and 2) ensuring that your event's content is accessible to a diverse audience.
+
+Firstly, consider how you will support speakers in developing accessible and inclusive material. 
+This may mean developing and sharing guidance on session content, or could also be giving feedback on presentation drafts. 
+Ensure that you contact and confirm your speakers with enough notice that allows them to prepare and revise material. 
+Please refer to the {ref}`cl-inclusive-events-comms-accessibility-guide` section for more recommendations on what to include in a speaker guidance pack for how to make presentations accessible. 
+
+Secondly, it is best practice to send out content before the session or event to ensure that people with sensory impairments, or who may require more time to review and engage with the material, are able to understand and take part. 
+
+If this is not possible, please address the reasons why in the opening address of the event and make alternative arrangements.
+For example, if the information is sensitive and unable to be shared due to Intellectual Property or Privacy reasons. 
+
+(cl-inclusive-events-comms-event-content)=
+### Collaborative Event Content
+If content such as presentation slides, worksheets, and other material are able to be shared but was not sent out before the event, consider sending it as soon as possible after the session. 
+You can then also send out notes or session minutes taken during the event.
+You could also consider taking collaborative notes during the session and making the document link available for attendees. 
+[HackMD](https://hackmd.io) or [Google Docs](https://www.google.co.uk/docs/about/) are recommended for their accessibility support features, and Etherpad can support a larger number of contributors. 
```

Comment:
I thought we were using Etherpad because it was more accessible? 

Maybe also a link to Etherpad!